HelpSource/Classes/Set.schelp

```diff
@@ -74,6 +75,7 @@ a -- b // shorter syntax
 
 method::isSubsetOf
 Returns true if all elements of this are also elements of strong::that::.
+Checks for equal elements regardless of the order inside the set.
```

Comment:
Set is unordered. While I think it doesn't hurt to state that again, maybe it should be in a form that supports a more solid knowledge of this important property?
Example:
"Checks for equality (not identity). Since Set is an unordered collection, order doesn't matter in this comparison"

Reply:
Sounds good. I just took your sentence.